src/main.cpp

```diff
@@ -1988,40 +1985,43 @@ bool CheckProofOfResearch(
         return true;
 
     // 6-4-2017 - Verify researchers stored block magnitude
-    double dNeuralNetworkMagnitude = CalculatedMagnitude2(bb.cpid, block.nTime, false);
-    if( bb.Magnitude > 0
-        && (fTestNet || (!fTestNet && pindexPrev->nHeight > 947000))
-        && bb.Magnitude > (dNeuralNetworkMagnitude*1.25) )
+    double dNeuralNetworkMagnitude = CalculatedMagnitude2(cpid, block.nTime);
+
+    if (claim.m_magnitude > (dNeuralNetworkMagnitude*1.25)
```

Comment:
We need to seriously think about getting rid of the 1.25 slop factor here in bv11.